spec/rails_helper.rb

```diff
@@ -1,41 +1,67 @@
-# This file is copied to spec/ when you run 'rails generate rspec:install'
-require 'spec_helper'
-ENV['RAILS_ENV'] ||= 'test'
-require File.expand_path('../config/environment', __dir__)
-# Prevent database truncation if the environment is production
-abort("The Rails environment is running in production mode!") if Rails.env.production?
-require 'rspec/rails'
-require 'shoulda/matchers'
-require 'factory_bot_rails'
-
-Dir[Rails.root.join('spec', 'support', '**', '*.rb')].sort.each { |f| require |f| }
-
-begin
-  ActiveRecord::Migration.maintain_test_schema!
-rescue ActiveRecord::PendingMigrationError => e
-  puts e.to_s.strip
-  exit 1
-end
-
-RSpec.configure do |config|
-  config.use_transactional_fixtures = false
-  config.infer_spec_type_from_file_location!
-  config.filter_rails_from_backtrace!
-
-  config.include FactoryBot::Syntax::Methods
-
-  config.before(:suite) do
-    DatabaseRewinder.clean_all
-  end
-
-  config.after(:each) do
-    DatabaseRewinder.clean
-  end
-end
-
-Shoulda::Matchers.configure do |config|
-  config.integrate do |with|
-    with.test_framework :rspec
-    with.library :rails
-  end
-end
+# This file is copied to spec/ when you run 'rails generate rspec:install'
```

Comment:
Do not delete the old code, you might need them in the future.

Reply:
updated